dask_ml/model_selection/_hyperband.py

```diff
@@ -0,0 +1,580 @@
+        subset of ``history_`` and ``model_id`` are model identifiers.
+
+    history_ : list of dicts
+        Information about each model after each ``partial_fit`` call. Each dict
+        the keys
+
+        * ``partial_fit_time``
+        * ``score_time``
+        * ``score``
+        * ``model_id``
+        * ``params``
+        * ``partial_fit_calls``
+
+        The key ``model_id`` corresponds to the ``model_id`` in ``cv_results_``.
+        This list of dicts can be imported into Pandas.
+
+    best_estimator_ : BaseEstimator
+        The model with the highest validation score among all the models
+        retained by the "inverse decay" algorithm.
+
+    best_score_ : float
+        Score achieved by ``best_estimator_`` on the vaidation set after the
+        final call to ``partial_fit``.
+
+    best_index_ : int
+        Index indicating which estimator in ``cv_results_`` corresponds to
+        the highest score.
+
+    best_params_ : dict
+        Dictionary of best parameters found on the hold-out data.
+
+    scorer_ :
+        The function used to score models, which has a call signature of
+        ``scorer_(estimator, X, y)``.
+
+    n_splits_ : int
+        Number of cross validation splits.
+
+    multimetric_ : bool
+        Whether this cross validation search uses multiple metrics.
+
+    Notes
+    -----
+    To set ``max_iter`` and the chunk size for ``X`` and ``y``, it is required
+    to know
+
+    * how many "epochs" or "passes through the data ``X``" to train the model
+      for (``epochs`` below)
+    * a rough idea of how many hyper-parameter combinations to sample
+      (``params_to_sample`` below)
+
+    To determine the chunk size and ``max_iter``,
+
+    1. Let the chunks size be ``chunk_size = epochs * len(X) / params_to_sample``
+    2. Let ``max_iter = params_to_sample``
+
+    where ``len(X)`` is the number of examples. Then, every estimator sees no
+    more than ``max_iter * chunk_size = len(X) * epochs`` examples.
+    Hyperband will actually sample some more hyper-parameter combinations,
+    so a rough idea of parameters to sample works.
+
+    For instance, let's say about 200 or 300 hyper-parameters need to be tested
+    to effectively search the possible hyper-parameters and estimators need a
+    little less than 100 epochs to converge. This will mean the dataset should
+    have chunks that are about 1/3rd of the entire dataset, and ``max_iter`` is
+    between 200 and 300.
+
+    If the search space is larger or more complex, choose to evaluate more
+    hyper-parameters. Increase the parameters to sample by a factor of 2 by:
+
+    * increasing ``params_to_sample`` by a factor of 2
+    * decreasing ``chunk_size`` by a factor of 2.
+
+    There are some limitations to the `current` implementation of Hyperband:
+
+    1. The full dataset is requested to be in distributed memory
+    2. The testing dataset should fit in the memory of a single worker
```

Comment:
The second point is mentioned above. Mirror the IncrementalSearchCV docs on the other one.